src/plugins/kibana/public/settings/sections/indices/add_data_steps/pipeline_setup/source_data.txt

```diff
@@ -0,0 +1,27 @@
+64.242.88.10
```

Comment:
Another convenience file. This is what populates the source_data dropdown at the top of the pipeline_setup. Can be removed as soon as step 1 produces input.